test error with vispy 0.7 (deprecated colormap)

## 🐛 Bug

In the CI for #3006, I am getting the error below. It looks like passing 'cubehelix' as a string to the vispy colormap is deprecated with `vispy >= 0.7`.

```python
 =================================== FAILURES ===================================
  _______________________ test_add_image_colormap_variants _______________________
  
      def test_add_image_colormap_variants():
          """Test adding image with all valid colormap argument types."""
          viewer = ViewerModel()
          np.random.seed(0)
          data = np.random.random((10, 15))
          # as string
          assert viewer.add_image(data, colormap='green')
      
          # as string that is valid, but not a default colormap
  >       assert viewer.add_image(data, colormap='cubehelix')
  
  napari/components/_tests/test_viewer_model.py:50: 
  _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
  napari/components/viewer_model.py:644: in add_image
      colormap = ensure_colormap(colormap)
  napari/utils/colormaps/colormap_utils.py:516: in ensure_colormap
      cmap = vispy_or_mpl_colormap(
  napari/utils/colormaps/colormap_utils.py:389: in vispy_or_mpl_colormap
      cmap = get_colormap(name)
  _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
  
  name = 'cubehelix', args = (), kwargs = {}
  cmap = <vispy.color.colormap.CubeHelixColormap object at 0x7f1c4fd973a0>
  
      def get_colormap(name, *args, **kwargs):
          """Obtain a colormap.
      
          Parameters
          ----------
          name : str | Colormap
              Colormap name. Can also be a Colormap for pass-through.
          *args:
              Deprecated.
          **kwargs
              Deprecated.
      
          Examples
          --------
              >>> get_colormap('autumn')
              >>> get_colormap('single_hue')
      
          .. versionchanged: 0.7
      
              Additional args/kwargs are no longer accepted. Colormap classes are
              no longer created on the fly. To create a ``cubehelix``
              (``CubeHelixColormap``), ``single_hue`` (``SingleHue``), ``hsl``
              (``HSL``), ``husl`` (``HSLuv``), ``diverging`` (``Diverging``), or
              ``RdYeBuCy`` (``RedYellowBlueCyan``) colormap you must import and
              instantiate it directly from the ``vispy.color.colormap`` module.
      
          """
          if args or kwargs:
              warnings.warn("Creating a Colormap instance with 'get_colormap' is "
                            "no longer supported. No additional arguments or "
                            "keyword arguments should be passed.", DeprecationWarning)
          if isinstance(name, BaseColormap):
              return name
      
          if not isinstance(name, str):
              raise TypeError('colormap must be a Colormap or string name')
          if name in _colormaps:  # vispy cmap
              cmap = _colormaps[name]
              if name in ("cubehelix", "single_hue", "hsl", "husl", "diverging", "RdYeBuCy"):
  >               warnings.warn(
                      f"Colormap '{name}' has been deprecated since vispy 0.7. "
                      f"Please import and create 'vispy.color.colormap.{cmap.__class__.__name__}' "
                      "directly instead.",
                      DeprecationWarning,
                      stacklevel=2,
                  )
  E               DeprecationWarning: Colormap 'cubehelix' has been deprecated since vispy 0.7. Please import and create 'vispy.color.colormap.CubeHelixColormap' directly instead.
  
  /tmp/.tox/py39-linux-pyside/lib/python3.9/site-packages/vispy/color/colormap.py:1135: DeprecationWarning
```
## To Reproduce

Steps to reproduce the behavior:

1. Run the viewer tests


## Expected behavior
We should not use the deprecated behavior

## Environment
From the CI
```
headless-py38-linux installed: alabaster==0.7.12,appdirs==1.4.4,asciitree==0.3.3,attrs==21.2.0,Babel==2.9.1,backcall==0.2.0,cachey==0.2.1,certifi==2021.5.30,charset-normalizer==2.0.1,cloudpickle==1.6.0,colorama==0.4.4,coverage==5.5,cycler==0.10.0,dask==2021.7.0,debugpy==1.3.0,decorator==4.4.2,docstring-parser==0.8.1,docutils==0.17.1,EasyProcess==0.3,fasteners==0.16.3,freetype-py==2.2.0,fsspec==2021.7.0,HeapDict==1.0.1,hsluv==5.0.2,hypothesis==6.14.2,idna==3.2,imageio==2.9.0,imagesize==1.2.0,iniconfig==1.1.1,ipykernel==6.0.1,ipython==7.25.0,ipython-genutils==0.2.0,jedi==0.18.0,Jinja2==3.0.1,jsonschema==3.2.0,jupyter-client==6.1.12,jupyter-core==4.7.1,kiwisolver==1.3.1,locket==0.2.1,magicgui==0.2.10,MarkupSafe==2.0.1,matplotlib==3.4.2,matplotlib-inline==0.1.2,napari==0.1.dev1+g435fcc9,napari-console==0.0.3,napari-plugin-engine==0.1.9,napari-svg==0.1.5,networkx==2.5.1,numcodecs==0.8.0,numpy==1.21.0,numpydoc==1.1.0,packaging==21.0,pandas==1.3.0,parso==0.8.2,partd==1.2.0,pexpect==4.8.0,pickleshare==0.7.5,Pillow==8.3.1,Pint==0.17,pluggy==0.13.1,pooch==1.4.0,prompt-toolkit==3.0.19,psutil==5.8.0,ptyprocess==0.7.0,py==1.10.0,pydantic==1.8.2,Pygments==2.9.0,PyOpenGL==3.1.5,pyparsing==2.4.7,pyrsistent==0.18.0,pytest==6.2.4,pytest-cov==2.12.1,pytest-faulthandler==2.0.1,pytest-order==1.0.0,pytest-qt==4.0.2,pytest-timeout==1.4.2,pytest-xvfb==2.0.0,python-dateutil==2.8.2,pytz==2021.1,PyVirtualDisplay==2.2,PyWavelets==1.1.1,PyYAML==5.4.1,pyzmq==22.1.0,qtconsole==5.1.1,QtPy==1.9.0,requests==2.26.0,ruamel.yaml==0.17.10,ruamel.yaml.clib==0.2.6,scikit-image==0.18.2,scipy==1.7.0,semgrep==0.57.0,six==1.16.0,snowballstemmer==2.1.0,sortedcontainers==2.4.0,Sphinx==4.1.0,sphinxcontrib-applehelp==1.0.2,sphinxcontrib-devhelp==1.0.2,sphinxcontrib-htmlhelp==2.0.0,sphinxcontrib-jsmath==1.0.1,sphinxcontrib-qthelp==1.0.3,sphinxcontrib-serializinghtml==1.1.5,superqt==0.2.1,tensorstore==0.1.12,tifffile==2021.7.2,toml==0.10.2,toolz==0.11.1,torch==1.9.0,tornado==6.1,tqdm==4.61.2,traitlets==5.0.5,typing-extensions==3.10.0.0,urllib3==1.26.6,vispy==0.7.1,wcwidth==0.2.5,wrapt==1.12.1,xarray==0.18.2,zarr==2.8.3
```

## Additional context
See CI run: https://github.com/napari/napari/pull/3006/checks?check_run_id=3064851061

--- a/napari/components/_tests/test_viewer_model.py
+++ b/napari/components/_tests/test_viewer_model.py
@@ -47,7 +47,7 @@ def test_add_image_colormap_variants():
     assert viewer.add_image(data, colormap='green')
 
     # as string that is valid, but not a default colormap
-    assert viewer.add_image(data, colormap='cubehelix')
+    assert viewer.add_image(data, colormap='fire')
 
     # as tuple
     cmap_tuple = ("my_colormap", Colormap(['g', 'm', 'y']))

